ros::Time time
float64[16] joint_pos
float64[16] joint_load
